packages/mask/src/plugins/Wallet/apis/debank.ts

```diff
@@ -29,14 +29,16 @@ export async function getAssetsList(address: string) {
 }
 
 const auroraChainOnDebank = getDeBankConstants(ChainId.Aurora).CHAIN_ID
+const harmonyChainOnDebank = getDeBankConstants(ChainId.Harmony).CHAIN_ID
+const arbitrumChainOnDebank = getDeBankConstants(ChainId.Arbitrum).CHAIN_ID
 const fieldKeys = ['fast', 'normal', 'slow'] as const
 /**
  * Debank's data might be outdated, like gas price for aurora which requires 1 Gwei at least
  * https://twitter.com/AlexAuroraDev/status/1490353255817302016
  * Once debank fixes it, we will remove this modifier.
  */
 function gasModifier(gasDict: GasPriceDictResponse, chain: string) {
-    if (chain === auroraChainOnDebank) {
+    if (chain === auroraChainOnDebank || harmonyChainOnDebank || arbitrumChainOnDebank) {
```

Comment:
Do you mean `[auroraChainOnDebank, harmonyChainOnDebank, arbitrumChainOnDebank].include(chain)`?